tests/integration_tests/test_cli.py

```diff
@@ -177,40 +177,42 @@ def test_train_cli_horovod(csv_filename):
         )
 
 
-@pytest.mark.distributed
-def test_export_savedmodel_cli(csv_filename):
-    """Test exporting Ludwig model to Tensorflows savedmodel format."""
-    with tempfile.TemporaryDirectory() as tmpdir:
-        config_filename = os.path.join(tmpdir,
-                                       'config.yaml')
-        dataset_filename = _prepare_data(csv_filename,
-                                         config_filename)
-        _run_ludwig('train',
-                    dataset=dataset_filename,
-                    config_file=config_filename,
-                    output_directory=tmpdir)
-        _run_ludwig('export_savedmodel',
-                    model_path=os.path.join(tmpdir, 'experiment_run', 'model'),
-                    output_path=os.path.join(tmpdir, 'savedmodel')
-                    )
-
-
-@pytest.mark.distributed
-def test_export_neuropod_cli(csv_filename):
-    """Test exporting Ludwig model to neuropod format."""
-    with tempfile.TemporaryDirectory() as tmpdir:
-        config_filename = os.path.join(tmpdir,
-                                       'config.yaml')
-        dataset_filename = _prepare_data(csv_filename,
-                                         config_filename)
-        _run_ludwig('train',
-                    dataset=dataset_filename,
-                    config_file=config_filename,
-                    output_directory=tmpdir)
-        _run_ludwig('export_neuropod',
-                    model_path=os.path.join(tmpdir, 'experiment_run', 'model'),
-                    output_path=os.path.join(tmpdir, 'neuropod')
-                    )
+# TODO(https://github.com/ludwig-ai/ludwig/projects/3#card-71743513): Re-enable.
```

Comment:
I tried running the `test_cli.py::test_collect_summary_activations_weights_cli` test case from your branch.   Received this error
```
root@d9923e5401f5:/opt/project/sandbox/pytest_area# pytest /opt/project/tests/integration_tests/test_cli.py::test_collect_summary_activations_weights_cli
ImportError while loading conftest '/opt/project/tests/conftest.py'.
../../tests/conftest.py:22: in <module>
    from ludwig.hyperopt.run import hyperopt
../../ludwig/hyperopt/run.py:9: in <module>
    from ludwig.hyperopt.execution import RayTuneExecutor, executor_registry
../../ludwig/hyperopt/execution.py:13: in <module>
    from ludwig.api import LudwigModel
../../ludwig/api.py:40: in <module>
    from ludwig.backend import Backend, initialize_backend
../../ludwig/backend/__init__.py:18: in <module>
    from ludwig.backend.base import Backend, LocalBackend
../../ludwig/backend/base.py:24: in <module>
    from ludwig.models.predictor import Predictor
../../ludwig/models/predictor.py:14: in <module>
    from ludwig.models.ecd import ECD
../../ludwig/models/ecd.py:7: in <module>
    from ludwig.features.feature_registries import input_type_registry, \
../../ludwig/features/feature_registries.py:18: in <module>
    from ludwig.features.audio_feature import AudioFeatureMixin, AudioInputFeature
../../ludwig/features/audio_feature.py:27: in <module>
    from ludwig.features.sequence_feature import SequenceInputFeature
../../ludwig/features/sequence_feature.py:26: in <module>
    from ludwig.features.base_feature import InputFeature
../../ludwig/features/base_feature.py:31: in <module>
    from ludwig.modules.metric_modules import METRICS_INPUTS_REGISTRY
../../ludwig/modules/metric_modules.py:22: in <module>
    from torchmetrics import Accuracy, AUROC, AverageMeter, IoU, MeanAbsoluteError,\
E   ImportError: cannot import name 'AverageMeter' from 'torchmetrics' (/usr/local/lib/python3.8/site-packages/torchmetrics/__init__.py)
```
May not be an issue with your code but looks like you may be missing some recent changes in other parts of Ludwig.  See this [slack thread](https://ludwig-ai.slack.com/archives/C01PFQE2E9J/p1635504356088100).  You may want to try an upstream merge.  

Other than this, the code changes seem reasonable given the context you wrote.

Reply:
Not an issue with my code, but it looks like the test error gets fixed with the suggestion on the slack thread you linked (refreshing requirements.txt).